Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 resolver = "2"
-members = ["build", "contracts", "ffi", "ffi/guests", "steel"]
+members = ["aggregation", "aggregation/guest", "build", "contracts", "ffi", "ffi/guests", "steel"]
```

Comment:
I think that's fair 👍 Let's do that in another PR to avoid touching all the files in this one. I can open that this week, if you don't get to it first.